src/containers/Layout/Layout.tsx

```diff
@@ -114,6 +114,7 @@ const Layout = () => {
   const closeUserMenu = useCallback(() => useUIStore.setState({ userMenuOpen: false }), []);
   const openLanguageMenu = useCallback(() => useUIStore.setState({ languageMenuOpen: true }), []);
   const closeLanguageMenu = useCallback(() => useUIStore.setState({ languageMenuOpen: false }), []);
+  const pageTitle = `${t('home')} - ${siteName}`;
```

Comment:
Is this an a11y requirement? I do not like putting `home` before the brand name 😬 .

Reply:
Preferably the homepage title should be unique and descriptive. Only putting the brand name in the title like "Symphony" or "beIN Sports" doesn't provide the user with a lot of context strictly speaking. We could risk a potential a11y failure here on a audit.

Though I agree it is not really pretty.

Though only having the brand name in the title is a practice that often occurs. So I think we can better accept it and wait that an a11y auditor have to say about it.

My preferred solution is defining a unique homepage title per client. Maybe through the JW appconfig? We should also take SEO (a little bit) in consideration, because "Homepage - Symphony" is not really something clients wants to see in the SERP. Also it doesn't contain any good keywords.

Reply:
`Browse - Symphony`?

Reply:
These are the most used patterns:

- Brand name -> The brand name is already known by most people, but it's not very useful
- Slogan -> e.g., `Watch and stream all your favorite live sports on beIN SPORTS`

Reply:
I think for now keep it simple and only use the brand name (how it was). 
A slogan is the preferred solution but probably not a quickwin.